src/github.com/getlantern/flashlight/genconfig/cloud.yaml.tmpl

```diff
@@ -16,13 +16,13 @@ client:
     maxmasquerades: 20
     qos: 2
     weight: 80
-  proxiedsites:
-    cloud:{{range $domain, $bool := .proxiedsites }}
-    - {{$domain}}{{end}}
   masqueradesets:
     cloudflare:{{range .masquerades}}
     - domain: {{.Domain}}
       ipaddress: {{.IpAddress}}{{end}}
+proxiedsites:
```

Comment:
This was the fix for getlantern/lantern#2276.  I had moved the proxiedsites configuration out of the client section, but forgot to update the template.  I've uploaded the fixed yaml file to Amazon already.